lxc/export.go

```diff
@@ -109,7 +110,7 @@ func (c *cmdExport) Run(cmd *cobra.Command, args []string) error {
 
 	// Get name of backup
 	backupName := strings.TrimPrefix(op.Get().Resources["backups"][0],
-		"/1.0/backups/")
+		fmt.Sprintf("/%s/backups/", version.APIVersion))
```

Comment:
Although this particular example is safe to do, I would prefer as not to be manually crafting URLs anymore. 
It leaves the opportunity for mishandling URL escaping in the future.
Instead URLs should be generated using `api.NewURL().Path(version.APIVersion, "backups").String()` where possible.